src/content/config.ts

```diff
@@ -0,0 +1,13 @@
+// Astro expects a content/config.ts file, therefore we export the collections here.
```

Comment:
Add a `docs/content-collections.md` (or somewhere else) to explain this whole new concept that where introducing? (or in a separate PR, also with plop files, etc)